src/meson.build

```diff
@@ -0,0 +1,50 @@
+# We create symlinks to the native extensions in the source tree for the tests
+# and for development. Meson does not allow you to build in-place and Python
+# cannot create a single namespace from two identically-named paths.
+
+libc_ext = py.extension_module(
+    'libc',
+    'portage_util_libc.c',
+    dependencies : py.dependency(),
+    subdir : 'portage' / 'util',
+    install : true
+)
+
+whirlpool_ext = py.extension_module(
+    '_whirlpool',
+    'portage_util__whirlpool.c',
+    dependencies : py.dependency(),
+    subdir : 'portage' / 'util',
+    install : true
+)
+
+run_command(
```

Comment:
I like this workaround! It probably warrants a comment with an explanation though.

Reply:
Thanks! I'm not sure what to say about it beyond what I said at the top of the file though. Any suggestions?

Reply:
Oh, I somehow overlooked that. Nevermind!